.evergreen.yml

```diff
@@ -67,7 +67,16 @@ functions:
       params:
         working_dir: spruce/evergreen
         command: make configure-mongod
-
+    - command: shell.exec
+      type: setup
+      params:
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
+          curl -LO https://fastdl.mongodb.org/tools/db/mongodb-database-tools-ubuntu2204-x86_64-100.7.5.tgz;
+          tar -zxvf mongodb-database-tools-ubuntu2204-x86_64-100.7.5.tgz
+          cd mongodb-database-tools-ubuntu2204-x86_64-100.7.5/bin
+          sudo cp -r * /usr/local/bin/
```

Comment:
Don't use root and we shouldn't be writing to this directory since this host is shared across different tasks and I believe evergreen only cleans up the working directory so this directory will not be cleaned up.